File: testcases/x86_testcases/test2.py
def main() -> None:
    """
    Compute the factorial of a number and print the same
    """
    x : int = 10
    i : int = 0
    for i in range(x + 1):
        j : int = 0
        val : int = 1
        for j in range(i):
            val = val * (j + 1)
        print("Value of factorial of ", i, " is equal to ", val)
    return

if __name__ == "__main__":
    main()

File: testcases/x86_testcases/x86_output/test2/test2.c
#include <stdio.h>

void main2() {
    int x = 10;
    for(int i=0; i<(x+1); i++) {
        int val = 1;
        for(int j=0;j<i;j++) val = (val*(j+1));
        printf("Value of factorial of ");
        printf("%d", i);
        printf(" is equal to ");
        printf("%d", val);
        printf("\n");
    }
    return;
}

int main() {
    main2();
    return 0;
}
